### PR TITLE
Make metrics calculation its own file

### PR DESCRIPTION
**Pull request by EKMeyerson**
This makes the metrics calculation clear. The biggest change is adding a `scoring` module under `covid_xprize`. 

Next steps:
- Pull out additional functions to use to update the metrics exploration at the end of the predictor robojudge notebook.
- Use these functions in the judging module
- Add tests for the metrics calculation (They will be easier to test now)

### CODE REVIEW

File: predictor_robojudge.ipynb [hunk 1]
```diff
@@ -182,7 +182,7 @@
     "    actual_df = actual_df[(actual_df.Date >= start_date_for_diff) & (actual_df.Date <= end_date)]\n",
     "    # Add GeoID column that combines CountryName and RegionName for easier manipulation of data\n",
     "    # np.where usage: if A then B else C\n",
```
Comment (EKMeyerson) [inline predictor_robojudge.ipynb:184]: Check for '' instead of nan to be consistent with judging module

File: predictor_robojudge.ipynb [hunk 2]
```diff
@@ -204,6 +204,18 @@
     "actual_df = get_actual_cases(latest_df, start_date, end_date)"
    ]
   },
```
Comment (EKMeyerson) [inline predictor_robojudge.ipynb:206]: Add population column

File: predictor_robojudge.ipynb [hunk 3]
```diff
@@ -427,34 +439,6 @@
     "        print(errors)"
    ]
   },
```
Comment (EKMeyerson) [inline predictor_robojudge.ipynb:441]: Use new function instead of these individual cells

Comment (ofrancon) [reply predictor_robojudge.ipynb:441]: 👍

**Review by ofrancon — APPROVED** [review]
lgtm

**Pull request merged**